### DISCUSSION TITLE
Convert Event Time to Local Time

### DISCUSSION THREAD

**Discussion by 13igred** [theOehrly/Fast-F1#252] [Q&A] [answered]
Hello,
I've noticed that the event times are all in local time - is there a UTC or standard reference time that can be used to convert to local time?

Thanks

**Comment by theOehrly** [comment, marked as answer]
Nope, sorry, there currently isn't. But this really should be added.

**Reply by 13igred** [reply]
Thank for the reply, I'll figure out something with the country location then.

**Reply by theOehrly** [reply]
Just FYI, this will be in the next release
